pkg/controller/argocd/util.go

```diff
@@ -799,16 +799,33 @@ func (r *ReconcileArgoCD) deleteClusterResources(cr *argoprojv1a1.ArgoCD) error
 }
 
 func (r *ReconcileArgoCD) removeManagedByLabelFromNamespace(namespace string) error {
```

Comment:
should we rename this function to `removeManagedByLabelFromNamespaces` ?

Reply:
done! update the name of the function